ouroboros-network/src/Ouroboros/Network/NodeToNode/Version.hs

```diff
@@ -45,6 +45,10 @@ data NodeToNodeVersion
     -- ^ Changes:
     --
     -- * Enable full duplex connections.
+    | NodeToNodeV_11
```

Comment:
There was some discussion today with @coot and @dcoutts that perhaps we don't need to add an era here.

- Once the HF happens, nodes that haven't updated will disconnect anyway.

- However, during the interval after us releasing the software but before the HF happening, Duncan has a some recollection of some wire-protocols being affected by the number of eras. This new version would prevent that.

Reply:
Talked to Marcin:

- I should "steal" V_10 for Conway, and change the network code to use V_11 for full duplex instead.

Reply:
^^^ This is now what this PR does

Reply:
However, see my other comment about _not doing_ this in light of Conway having been delayed.